Website/src/rendering/src/components/HeroSectionCta.tsx

```diff
@@ -0,0 +1,18 @@
+import { ComponentProps } from 'lib/component-props';
+import { LinkField, Link } from '@sitecore-jss/sitecore-jss-nextjs';
+
+export type HeroCtaProps = ComponentProps & {
```

Comment:
This type name should match the component: `HeroSectionCtaProps`

Reply:
Updated